app/src/main/java/com/example/triptracker/view/profile/UserProfileOverview.kt

```diff
@@ -22,33 +22,60 @@ import androidx.compose.material3.Icon
 import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
 import androidx.compose.ui.*
 import androidx.compose.ui.draw.clip
 import androidx.compose.ui.draw.shadow
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import androidx.navigation.compose.rememberNavController
 import coil.compose.AsyncImage
 import com.example.triptracker.R
 import com.example.triptracker.view.Navigation
+import com.example.triptracker.view.NavigationBar
 import com.example.triptracker.view.Route
 import com.example.triptracker.view.TopLevelDestination
-import com.example.triptracker.viewmodel.UserProfileViewModel
+import com.example.triptracker.view.theme.md_theme_dark_gray
+import com.example.triptracker.view.theme.md_theme_light_dark
+import com.example.triptracker.view.theme.md_theme_orange
 
 /* Visual preview without any logic */
 @Preview
 @Composable
 fun UserProfilePreview() {
+  val navController = rememberNavController()
+  val navigation = remember(navController) { Navigation(navController) }
+  UserProfileOverview(navigation = navigation)
+}
+/**
+ * Composable displaying the profile overview featuring information about follower, following. 4
+ * clickable buttons are available to access to the MyTrips, Favorites, Friends and settings views.
+ *
+ * @param userProfileViewModel: The view model of the UserProfile
+ * @param navigation: The navigation of the app to switch between different views
+ */
+@Composable
+fun UserProfileOverview(
+    // userProfileViewModel: UserProfileViewModel = UserProfileViewModel(),
+    navigation: Navigation
+) {
+  // UID of the user that needs to be fetched from ??, used to fetch the profile from the database
+  // val uid = "TODO"
+  // val profile = userProfileViewModel.userProfileList.getUserProfile(uid)
+  // val url = profile.profileImageUrl

```

Comment:
Are these a work in progress/ need to be implemented? If so we need to add to the Backlog.

Reply:
These are code that will be used later when the Profil will be linked to the UserProfile viewModel